src/main/java/appeng/client/render/effects/LightningFX.java

```diff
@@ -35,6 +35,7 @@
 
 	private static final Random RANDOM_GENERATOR = new Random();
 	private static final int STEPS = 5;
+	private static final int BRIGHTNESS = 13 << 4;
```

Comment:
It's actually not in this case. .lightmap( BRIGHTNESS, BRIGHTNESS ) already uses the unpacked format, but for *whatever* reason, it still wants the << 4.
Vanilla I guess *shrug*.